src/storage/AdminProcessor.h

```diff
@@ -20,7 +21,32 @@ class TransLeaderProcessor : public BaseProcessor<cpp2::AdminExecResp> {
     }
 
     void process(const cpp2::TransLeaderReq& req) {
-        UNUSED(req);
+        auto spaceId = req.get_space_id();
+        auto partId = req.get_part_id();
+        auto ret = kvstore_->part(spaceId, partId);
+        if (!ok(ret)) {
+            resp_.set_code(to(error(ret)));
+            promise_.setValue(std::move(resp_));
```

Comment:
You could call onFinished() method in BaseProcessor.h

Reply:
It seems the response of all method in AdminProcess.h returns AdminExecResp, which is different from other regular storage methods. Maybe we need to refactor BaseProcessor later.